### PR TITLE
Update logging flag to match recommended style and execution.

### PR DESCRIPTION
**Pull request by annzimmer**

### CODE REVIEW

File: firebase-ml-modeldownloader/api.txt [hunk 1]
```diff
@@ -61,7 +61,7 @@ package com.google.firebase.ml.modeldownloader {
     method @NonNull public com.google.android.gms.tasks.Task<com.google.firebase.ml.modeldownloader.CustomModel> getModel(@NonNull String, @NonNull com.google.firebase.ml.modeldownloader.DownloadType, @Nullable com.google.firebase.ml.modeldownloader.CustomModelDownloadConditions);
     method public long getModelDownloadId(@NonNull String);
     method @NonNull public com.google.android.gms.tasks.Task<java.util.Set<com.google.firebase.ml.modeldownloader.CustomModel>> listDownloadedModels();
-    method public void setStatsCollectionEnabled(boolean);
+    method public void setDataCollectionMlDownloaderState(boolean);
```
Comment (vkryachko) [inline firebase-ml-modeldownloader/api.txt:64]: I think the approved naming pattern is `set${product}CollectionEnabled(Boolean)`

cc @VinayGuthal to confirm

Comment (annzimmer) [reply firebase-ml-modeldownloader/api.txt]: Looking through - your recommendation is the format used by crashlytics and performance - so updated to match.

Comment (VinayGuthal) [reply firebase-ml-modeldownloader/api.txt]: @vkryachko is right.

File: firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java [hunk 2]
```diff
@@ -463,11 +463,12 @@ private void deleteModelDetails(@NonNull String modelName) {
...
 
   /**
-   * Update the settings which allow logging to firelog.
+   * Update the settings which allow logging to firelog. When not specifically set, defaults to use
+   * the Firebase wide data collection switch.
    *
    * @param enabled - is statistics logging enabled
    */
-  public void setStatsCollectionEnabled(boolean enabled) {
+  public void setMlDownloaderCollectionEnabled(boolean enabled) {
```
Comment (VinayGuthal) [inline firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java:471]: Please add a java boolean over here. We also need to support the clear functionality to handle all edge cases.

Comment (VinayGuthal) [inline firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java:471]: Please add a java boolean over here. We also need to support the clear functionality to handle all edge cases.

**Review by VinayGuthal — CHANGES_REQUESTED** [review]

File: firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java (hunk 2 above)
Comment (annzimmer) [reply firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloader.java]: done

File: firebase-ml-modeldownloader/api.txt (hunk 1 above)
Comment (annzimmer) [reply firebase-ml-modeldownloader/api.txt]: Done - updated method to setModelDownloaderCollectionEnabled(Boolean) and added null edges to reset to default firebase level data collection.

**Review by VinayGuthal — APPROVED** [review]

**Pull request merged**